README.md

```diff
@@ -76,10 +76,10 @@ Together, these third-party integrations make GHDL a comprehensive tool for digi
   - After each successful CI run, [nightly](https://github.com/ghdl/ghdl/releases/tag/nightly) tarballs/zipfiles for Ubuntu and Windows (MSYS2) are updated. These nightly builds provide access to the latest features and updates, which may not yet be available in the package manager repositories.
   - If you need to set up GHDL in a Continuous Integration (CI) pipeline, the [setup-ghdl-ci](https://github.com/ghdl/setup-ghdl-ci) GitHub Action allows you to configure GHDL with minimal effort. It provides a simple and efficient way to integrate GHDL into your CI workflows with just a few lines of configuration.
 
-- **Using Docker**:  
```

Comment:
@oscargus your editor doesn't use the correct settings from `.editorconfig`. It removed trailing spaces, which are a forced line break in Markdown. Please correct these removals.